parm/config/gfs/config.base

```diff
@@ -227,6 +230,8 @@ case "${APP}" in
     ;;
   ATMA)
     export DO_AERO="YES"
+    export AERO_ANL_CDUMP="gdas"
```

Comment:
I think we want the default ATMA behavior to be `AERO_ANL_CDUMP="both"` and `FCST="gdas"` if I understand these correctly. Basically, we want the analysis in both GFS and GDAS but only the GDAS fcst has aerosols in it.

Reply:
OK, I've turned it on for both and will test for the gfs cycle.

Reply:
I don't remember if we have "gfs" obs for the test or not, so if it fails, we can revert it back.

Reply:
It works!